### PR TITLE
Added link to share results

### PR DESCRIPTION
**Pull request by anusharanganathan**
This is a PR for https://github.com/antleaf/jct-project/issues/244

**WARNING**
**I have changed the value of the base url in the api_endpoint.js. 
This is also propagated to the file jct_plugin.js. 
When merging to master reset the value of the base url in both of these files.**

This is so that the links within test and prod point to their environments.

### CODE REVIEW

File: layouts/partials/head.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <script type="text/javascript" src="/js/feedback.js?v=2.0.0"></script>
     {{ if .IsHome }}
         <script type="text/javascript" src="/js/api_codes.js?v=2.0.0"></script>
+        <script type="text/javascript" src="/js/find_out_more.js?v=2.0.0"></script>
```
Comment (richard-jones) [inline layouts/partials/head.html:20]: @anusharanganathan is this file missing from the PR?

Comment (anusharanganathan) [reply layouts/partials/head.html:20]: @richard-jones no, it already exists. It was previously only used by the plugin. Now it's also used by the main site.

**Review by richard-jones — CHANGES_REQUESTED** [review]
Can I suggest a few changes to how this is presented, and then ask for an additional feature:

1. Could we move the "Share this result" to be a button on the same level as the "Explain this result".
2. When you click that button, can it open a modal with the URL in a textbox that the user can copy to their clipboard.  I think if it's presented as a link, it isn't obvious that you shouldn't click on it, but instead need to right click.  Example of how we've done this on DOAJ: https://doaj.org/search/journals (click on Share/Embed).
3. Could we modify the back-end so that this URL is also sent to Zendesk when a user submits their feedback, so it's quick for us to get back to the page the user was looking at

**Comment by anusharanganathan** [comment]
See PR https://github.com/CottageLabs/jct/pull/64.

**Pull request closed without merging**